packages/react/src/Card/Card.module.css

```diff
@@ -197,3 +197,15 @@
 .Card:has(.Card__link:hover) .Card--expandableArrow path:nth-of-type(2) {
   stroke-dashoffset: 20;
 }
+
+/* Remove all animations, transitions and smooth scroll for people that prefer not to see them */
+@media (prefers-reduced-motion: reduce) {
```

Comment:
Looking through the docs with prefers-motion enabled, Card feels a bit jarring as it jumps to a slightly bigger size.

Do you think we should remove the scale effect altogether? If we wanted to keep a hover style then maybe we could apply a focus ring on hover instead?

Reply:
> Card feels a bit jarring as it jumps to a slightly bigger size
> Do you think we should remove the scale effect altogether? 

Yeah, agree but would prefer we don't tackle that here. The scale effect is existing behaviour and unrelated to the global rule change here. The goal of this update is to have parity and not regress. 

I do remember various conversations about this before though 🤔, and a11y design confirmed this was okay. @joshfarrant did this come up in recent audits? I also think focus ring could be misleading because that relates to a different type of interaction; focus vs hover.

Reply:
No prob, that works for me 👍

No this hasn't come up at all. Purely my opinion that the jump in size feels a bit off to me